### PR TITLE
Use credentials 1189.vf61b_a_5e2f62e

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Use credentials 1189.vf61b_a_5e2f62e

- Bump credentials in /bom-weekly
- Pin credentials plugin for 2.332.x and 2.346.x

Includes https://github.com/jenkinsci/bom/pull/1465 and pins credenitals plugin at an older release.

Once this is merged, it should unblock https://github.com/jenkinsci/bom/pull/1479 and allow the Jenkins version in bom-weekly to increment from 2.371 to 2.376 or 2.377.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by MarkEWaite** [comment]
I don't know how to resolve the failure reported in MemoryCleanupTest when it cannot find a netbeans class https://github.com/jenkinsci/bom/pull/1564/checks?check_run_id=9360359712

**Comment by MarkEWaite** [comment]
@timja or @jglick could you merge this and the others that are passing in hopes that we can unblock #1479

**Comment by MarkEWaite** [comment]
That would be very nice.  I'm not sure that I can be much help, but if I can have commit access, I may be able to help sometimes.

**Comment by timja** [comment]
done

File: bom-2.332.x/pom.xml
Comment (jglick) [inline bom-2.332.x/pom.xml:32]: What is this for? It should be inheriting the version from the 2.346.x line.

**Comment by jglick** [comment]
Also https://github.com/jenkins-infra/interesting-category-action/issues/1

File: bom-2.332.x/pom.xml
Comment (MarkEWaite) [reply bom-2.332.x/pom.xml:32]: Thanks for the education.  I had forgotten that the 2.346.x line is taken into the 2.332.x line.  I'll remove that wasted addition in a later pull request.

Comment (jglick) [reply bom-2.332.x/pom.xml:32]: Thanks!

**Comment by jglick** [comment]
`dependencies` is what is needed. `java` does not matter, that is just added automatically by Dependabot also.

**Comment by jglick** [comment]
see https://github.com/jenkinsci/.github/blob/1a7c2b7a644bf6fd18aaaa31cc4ddf44f22bcd07/.github/release-drafter.yml#L9-L66

**Pull request merged**
